contracts/staking/contracts/src/vaults/StakingPoolRewardVault.sol

```diff
@@ -211,8 +255,8 @@ contract StakingPoolRewardVault is
         view
         returns (uint256)
     {
-        Balance memory balance = balanceByPoolId[poolId];
-        return balance.operatorBalance + balance.membersBalance;
+        Pool memory pool = poolById[poolId];
```

Comment:
Nit: We could probably make the same change to this as the above; however, we are really only gaining minimal efficiency, whereas in the above case it actually could be significant, so this one is up to you.